src/modules/facets/ui.js

```diff
@@ -114,7 +114,7 @@ function facetUIElem (facetObj, children, isExpanded,facetSearchTxt, facet) {
         viewMoreCss="UNX-view-more";
         let docRoot = document.querySelector(":root");
         docRoot.style.setProperty('--viewMoreHeight', `${32*viewMoreLimit}px`);
-        viewMoreUi = `<div class="UNX-view-more-row "><button class="${actionBtnClass}" data-facet-name="${facetName}" data-action="viewMore" data-id="${viewMoreText[0]}">${viewMoreText[0]}</button></div>`;
+        viewMoreUi = `<div class="UNX-view-more-row "><button class="${this.options.actionBtnClass || actionBtnClass}" data-facet-name="${facetName}" data-action="viewMore" data-id="${viewMoreText[0]}">${viewMoreText[0]}</button></div>`;
```

Comment:
First priority for actionBtnClass inside facet and then less priority for this.options.actionBtnClass. Same applied below also